### PR TITLE
Add CI for tilt

### PR DESCRIPTION
**Pull request by ncdc**
# Description

## Reviewer Checklist

- [ ] API Go Documentation
- [ ] Tests: Unit Tests (and E2E Tests, if appropriate)
- [ ] Comprehensive Commit Messages
- [ ] Links to related GitHub Issue(s)

### CODE REVIEW

File: .github/workflows/tilt.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+on:
+  pull_request:
+    paths:
```
Comment (m1kola) [inline .github/workflows/tilt.yaml:3]: Maybe `paths-ignore` is more suitable here?

**Review by m1kola — APPROVED** [review]
Wondering if `paths-ignore` will be easier to maintain than `paths`. Other than that - looks good to me.

File: .github/workflows/tilt.yaml (hunk 1 above)
Comment (ncdc) [reply .github/workflows/tilt.yaml:3]: I kept going back and forth on this. I can switch it if you'd prefer.

**Comment by ncdc** [comment]
Proceeding to merge given the approval. Can adjust the paths config if needed in a follow up.

**Pull request merged**